koku/masu/processor/tasks.py

```diff
@@ -69,8 +69,9 @@
 REFRESH_MATERIALIZED_VIEWS_QUEUE = "reporting"
 REMOVE_EXPIRED_DATA_QUEUE = "remove_expired"
 SUMMARIZE_REPORTS_QUEUE = "process"
-UPDATE_COST_MODEL_COSTS_QUEUE = "reporting"
+UPDATE_COST_MODEL_COSTS_QUEUE = "priority_update"
 UPDATE_SUMMARY_TABLES_QUEUE = "reporting"
+DELETE_SOURCE_QUEUE = "priority_delete"
```

Comment:
I think it would be good to simply have a `PRIORITY_QUEUE = "priority"` instead of different queues for the different highly prioritized tasks.